internal/parsers/table/csv/calls.go

```diff
@@ -79,16 +81,38 @@ func (t *Table) SubDivName(countryCode *locodedb.CountryCode, code string) (stri
 	return rec.name, nil
 }
 
+func utf8encoding(subdivCountry string) (string, error) {
+	res, err := charmap.Windows1256.NewDecoder().String(subdivCountry)
+	if err == nil && !strings.Contains(res, "\uFFFD") && !strings.Contains(res, "\u0000") && !strings.Contains(res, "?") {
+		return res, nil
+	}
+
+	res, err = charmap.ISO8859_1.NewDecoder().String(subdivCountry)
```

Comment:
I'd try this first and 1256 then. But maybe you're right, I don't know.

Reply:
tried this way, but charmap.ISO8859_1 leaves move unknown/ unrelevant symbols, and doesn't give any error on it..